lib/vips.rb

```diff
@@ -626,6 +626,10 @@ def to_s
   attach_function :vips_leak_set, [:int], :void
   attach_function :vips_vector_set_enabled, [:int], :void
   attach_function :vips_concurrency_set, [:int], :void
+  attach_function :vips_concurrency_get, [], :int
+
+  # Track the original default concurrency so we can reset to it.
+  DEFAULT_CONCURRENCY = vips_concurrency_get
```

Comment:
Read the default right away so we can reset to it. (Note `Vips.vips_init` just above which sets the initial `vips__concurrency` to `VIPS_CONCURRENCY` or CPU count.)